### PR TITLE
fix-trainee-dashboard

### PR DESCRIPTION
**Pull request by Charlesntwari**
# PR Description
this PR modify the designs dashboard of trainee so that it looks great than ever especially in dark mode

# Description of tasks that were expected to be completed
-double check and analyze  the logic used represent data in graphs
-modifying data representation designs in table format
-fixing all data on the screen properly

# How has this been tested?

-pull this PR
-run both server and frontend
-login as trainee and check on dashboard
(N.B: in case trainee doesn't have ratings for the sake of having better experience please first of all rate him/her as coordinator then back to check the changes)
# Number of Commits
1

# Screenshots (If appropriate)

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

### CODE REVIEW

File: src/components/ViewComment.tsx
Comment (nshutip) [inline src/components/ViewComment.tsx:1]: Why was the color code changed on line 58 from #333131 to #8667F2 on line 62?

**Review by nshutip — APPROVED** [review]

**Review by Virgile1k — APPROVED** [review]
checked and approved

File: src/components/ViewComment.tsx
Comment (Charlesntwari) [reply src/components/ViewComment.tsx:1]: that color button was not matching with our button's color code, moreover it wasn't looking good enough in the dark mode. and as point of information the line of codes are now fixed as it was. only color code was changed in that file.

**Review by NTElissa — APPROVED** [review]

**Review by ceelogre — CHANGES_REQUESTED** [review]
The phase element should retain the original color in light mode.
image

**Comment by Charlesntwari** [comment]
> The phase element should retain the original color in light mode. image

Normally based on the logic of the system phase element comes with trainee ratings. that's why it does not appear on the dashboard of trainee with no performance ratings

**Comment by ceelogre** [comment]
> > The phase element should retain the original color in light mode. image
> 
> Normally based on the logic of the system phase element comes with trainee ratings. that's why it does not appear on the dashboard of trainee with no performance ratings

I'm talking about the whole div element not particularly the phase.

**Review by ceelogre — APPROVED** [review]

**Pull request merged**